tests/app/organisation/test_rest.py

```diff
@@ -275,7 +275,8 @@ def test_post_create_organisation_existing_name_raises_400(admin_request, sample
             },
             (
                 "organisation_type foo is not one of "
-                "[central, local, nhs_central, nhs_local, nhs_gp, emergency_service, school_or_college, other]"
+                "[central, local, nhs_central, nhs_local, nhs_gp, emergency_service, school_or_college, "
+                "other]"
```

Comment:
Was this one of your auto-formatters kicking in or a manual choice? We leave most of our formatting decisions to `black` and linting to `ruff`.

If you haven't already: we provide a set of [pre-commit](https://pre-commit.com/) hooks that you can use in pretty much every relevant repo. Download/install it and then in each repo run `pre-commit install --install-hooks`. Then on every commit we'll run things like autoformatting, fixing line endings, and throwing linting errors etc.

Reply:
I think it was one of the autoformatters kicking off. I'll make sure precommit hooks are working.